bescort.lic

```diff
@@ -243,7 +249,17 @@ class Bescort
       pause 0.5
     end
   end
-
+	
+	def oshu_manor(mode) 
+		case mode
+		when /exit/i
+      walk_to(2317)
+    when /enter/i
+			wait_for_script_to_complete('oshu_manor')
+			find_room_list(%w(e e w w w n n n))
```

Comment:
http://pastebin.com/raw/NbkjVw5R

That's a log of what oshu_manor.lic does (when not running into trouble).